browser/src/control/Control.UIManager.js

```diff
@@ -790,6 +790,7 @@ L.Control.UIManager = L.Control.extend({
 		$('.main-nav').hide();
 		if (L.Params.closeButtonEnabled) {
 			$('#closebuttonwrapper').hide();
+			$('#closebuttonwrapperseparator').hide();
```

Comment:
you forgot to add also "show" in above function showMenubar

Reply:
it works here on mode switch, but it is needed to make this functions work properly if used outside of that UI switch (eg. postmessage) I think

Reply:
good catch, thanks!

Reply:
added `$('#closebuttonwrapperseparator').show();` to `showMenubar()` function